pilot/pkg/bootstrap/istio_ca.go

```diff
@@ -404,9 +374,10 @@ func (s *Server) createIstioCA(client corev1.CoreV1Interface, opts *caOptions) (
 	// ca.go saves or uses the secret, but also writes to the configmap "istio-security", under caTLSRootCert
 	// rootCertRotatorChan channel accepts signals to stop root cert rotator for
 	// self-signed CA.
-	rootCertRotatorChan := make(chan struct{})
-	// Start root cert rotator in a separate goroutine.
-	istioCA.Run(rootCertRotatorChan)
+	s.addStartFunc(func(stop <-chan struct{}) error {
+		go istioCA.Run(stop)
```

Comment:
not needed. Run starts its own go routine.

Reply:
good catch